tools/xdp/s2n-quic-xdp/src/socket.rs

```diff
@@ -28,6 +28,20 @@ impl Fd {
         let fd = Arc::new(Inner(fd));
         Ok(Self(fd))
     }
+
+    pub fn attach_umem(&self, umem: &crate::umem::Umem) -> Result<()> {
+        umem.attach(self)?;
+        // TODO store the umem
```

Comment:
you need an issue for this?

Reply:
Nah. I'm going to be addressing it with the next PRs